### PR TITLE
style: use black formatter

### PR DESCRIPTION
**Pull request by tobiichi3227**

### CODE REVIEW

**Comment by LifeAdventurer** [comment]
Please do the followings
- put all the `f"""` that is in `textwrap.dedent()` back.
- standardize all formats, such as if-else conditions, and with open statements.
- reverse indentation edits

**Comment by tobiichi3227** [comment]
I have some questions about your styling instruction.
First, what is the reverse indentation edits?
Second, the black formatter use pep8 standard, i don't know which statement need to standardize.
If you think 88 characters in single line (black default config) is too short, i can change this rule to 120 characters.

File: src/response_terminal.py [hunk 1]
```diff
@@ -639,40 +756,46 @@ def get_response_in_terminal_mode(message) -> str:
             # if msg > '6' or msg < '1':
             #     return 'no such command'
             # else:
-                return function_developing()
-            # TO-DO
-            # elif msg == 'setup':
+            return function_developing()
+        # TO-DO
```
Comment (LifeAdventurer) [inline src/response_terminal.py:736]: This is the first point.
This function is still developing, so please change it back.

File: src/response_terminal.py [hunk 2]
```diff
@@ -639,40 +756,46 @@ def get_response_in_terminal_mode(message) -> str:
...
+        # elif msg == 'setup':
```
Comment (LifeAdventurer) [inline src/response_terminal.py:738]: This is the first point.
This function is still developing, so please change it back.

Comment (tobiichi3227) [reply src/response_terminal.py]: Change in latest commit

**Comment by LifeAdventurer** [comment]
Now, I know this line break styling is based on line length.

**Comment by tobiichi3227** [comment]
> Now, I know this line break styling is based on line length.

Sorry, I should talk you first time.

**Pull request closed without merging**